docs/runbooks/CnaoNotReady.md

```diff
@@ -0,0 +1,28 @@
+# CnaoNotReady
+
+## Meaning
+
+Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
+This alert fires when one or more of CNAO deployed components is not ready.
+
+## Impact
+
+Without CNAO components deployed and ready, the user might lack certain network components
+that are provided by this component.
+
+## Diagnosis
+
+- Check for messages on the networkaddonsconfig CR status:
+	```
+	kubectl get networkaddonsconfig -o custom-columns="":.status.conditions[*].message
+	```
```

Comment:
Maybe extracting the deployment/daemonSet have to be here and also readin the logs.

Reply:
fixed.